docs/getting-started/getting-started-with-codacy.md

```diff
@@ -25,23 +23,29 @@ Now, you'll need to join one or more organizations that contain your repositorie
 
 To start adding your repositories, select one of the organizations and click **Go to add repositories**.
 
-## 4. Add repositories {: id="add-repositories"}
+## 3. Add repositories {: id="add-repositories"}
 
 As a final step, add the repositories that you wish to analyze. Codacy will start the first analysis and set up everything required to ensure your next commits on those repositories are analyzed.
```

Comment:
I know the scope of this PR is around IA *but* would it be worth mentioning here that only repo admins can add repositories? (I'd confirm the truthfulness of this for Bitbucket)

Reply:
Hello @inesbpinto! :wave: :smile: 

Don't worry, this pull request got a bit more complicated than what I was expecting, and I'll probably have to start from scratch after I finish making some other unrelated changes that also affect the IA a bit. :+1: 

Your suggestions do make a lot of sense! :raised_hands: I opened this issue #418 on your behalf and will tackle it very soon. :+1:

Reply:
@inesbpinto Just as a heads up, I opened a [separate pull request](https://github.com/codacy/docs/pull/428) to make the improvements to the "Getting started", separate from all the other IA changes which are a bit stuck at the moment. :)